hathor/mining/ws.py

```diff
@@ -129,7 +129,12 @@ def send_notification(self, *, method: str, params: Union[List, Dict]) -> None:
         }
         if params:
             request['params'] = params
-        self.sendMessage(json_dumpb(request))
+        try:
+            self.sendMessage(json_dumpb(request))
```

Comment:
As we need to capture `Exception`, I think we should move the serialization to outside the try/except block.

Reply:
Actually, I think we might want to capture that too, besides if the serialization fails it should be an implementation error because it does not rely on data that isn't already in a format we can serialize. The point is that if `send_notifications` fails we will miss a send for the rest of the connections on the `broadcast_notification` loop.

Reply:
I agree. Thanks!